apps/remix-ide/src/app/files/fileManager.js

```diff
@@ -57,6 +57,12 @@ class FileManager extends Plugin {
     this.mode = mode
   }
 
+  async compileWithHardhat (fileContent) {
+    const configFilePath = 'remixCompiler.config.js'
```

Comment:
maybe this should be `'remix-compiler.config.js'`

Reply:
Done